### PR TITLE
Relative target speed

### PR DESCRIPTION
**Pull request by kyabe2718**
## Types of PR

- [x] New Features
- [ ] Upgrade of existing features
- [ ] Bugfix

## Description
add support for RelativeTargetSpeed

## How to review this PR.
please check passing all test and new scenario (relative_target_speed.yaml)

### CODE REVIEW

**Comment by hakuturu583** [comment]
Convert to draft because of #358

**Review by hakuturu583 — CHANGES_REQUESTED** [review]
Code looks good to me, please add this changes to the ReleaseNote

**Review by yamacir-kit — APPROVED** [review]

File: docs/ReleaseNotes.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
 - Update interpreter to access TrafficSignals from Action / Condition
 - Update EgoEntity to use precise simulation model parameters
 - add getVehicleCommand function to the API class
+- Add support for RelativeTargetSpeed, the syntax of OpenSCENARIO
```
Comment (hakuturu583) [inline docs/ReleaseNotes.md:16]: It seems, this file is different from master branch.

**Review by hakuturu583 — CHANGES_REQUESTED** [review]

**Comment by hakuturu583** [comment]
I fixed relase notes!

**Review by hakuturu583 — APPROVED** [review]

**Pull request merged**